tests/playwright-test/playwright-test-fixtures.ts

```diff
@@ -48,7 +48,7 @@ export type RunResult = {
   interrupted: number,
   didNotRun: number,
   report: JSONReport,
-  results: any[],
+  results: JSONReportTestResult[],
```

Comment:
this leads to a type failure in `fixtures.spec.ts`

Reply:
extracted into https://github.com/microsoft/playwright/pull/34883